packages/wagmi/package.json

```diff
@@ -45,15 +45,18 @@
     "react": "^18.2.0",
     "react-dom": "^18.2.0",
     "typescript": "^4.9.5",
-    "wagmi": "^0.12.13"
+    "viem": "^1.0.0",
+    "wagmi": "^1.1.0"
   },
   "peerDependencies": {
     "@lens-protocol/react-web": "workspace:*",
     "ethers": "^5.7.2",
-    "wagmi": "0.12.7 - 0.12.13"
+    "viem": "^1.0.0",
+    "wagmi": "^1.1.0"
   },
   "prettier": "@lens-protocol/prettier-config",
   "dependencies": {
+    "@ethersproject/abstract-signer": "^5.7.0",
```

Comment:
we probably don't need it as as of now the whole `ethers` is a `peerDependency` anyway

Reply:
cool, thanks